apps/prairielearn/src/sync/sharing.ts

```diff
@@ -60,7 +60,14 @@ export function checkInvalidPublicSharingRemovals(
 
     // TODO: allow if question is not used in anyone else's assessments
     const questionData = courseData.questions[question.qid].data;
-    if (!(questionData?.sharedPublicly || questionData?.sharedSourcePublicly)) {
+    if (
+      !(
+        questionData?.sharePublicly ||
+        questionData?.sharedPublicly ||
+        questionData?.shareSourcePublicly ||
+        questionData?.sharedSourcePublicly
+      )
+    ) {
```

Comment:
Ok this is actually my fault. This if statement should actually just be 
```js
if (!(
        questionData?.sharePublicly ||
        questionData?.sharedPublicly))
```

I originially wrote this if statement back when we were going to say that shareSourcePublicly should imply sharePublcy, but then we decided that the two should actually operate independently.